scripts/prepdocs.py

```diff
@@ -314,6 +314,29 @@ def refresh_openai_token():
         openai.api_key = token_cred.get_token("https://cognitiveservices.azure.com/.default").token
         open_ai_token_cache[CACHE_KEY_CREATED_TIME] = time.time()
 
+"""Readfile function
+
+recursively read directory structure under `pathPattern` and execute indexing for the individual files
+"""
+def read_files(pathPattern, use_vectors):
```

Comment:
pathPattern should be path_pattern, per naming conventions detailed at pep8.org

Please also add a type annotation- learn more here: https://realpython.com/lessons/annotations/

Reply:
done